types/hedron/index.d.ts

```diff
@@ -1,172 +1,147 @@
-     * @default 'section'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * The amount of horizontal columns this row creates.
-     * @default 12
-     */
-    divisions?: number;
-    /**
-     * Sets the value of the CSS align-content property
-     */
-    alignContent?: string;
-    /**
-     * Sets the value of the CSS align-items property
-     */
-    alignItems?: string;
-    /**
-     * Sets the value of the CSS align-self property
-     */
-    alignSelf?: string;
-    /**
-     * Sets the value of the CSS justify-content property
-     */
-    justifyContent?: string;
-    /**
-     * Sets the value of the CSS order property
-     */
-    order?: string;
+export interface BoundsProps extends Styleable {
+  /**
+   * Draws all child columns with "bounding boxes" for easy
+   * visualization of the grid. This enables debug mode for all the
+   * children of this component
+   * @default false
+   */
+  debug?: boolean;
+  /**
+   * Controls the CSS flex property
+   */
+  flex?: string;
+  /**
+   * Direction of content
+   */
+  direction?: "horizontal" | "vertical";
+  /**
+   * Alignment of children along the vertical axis
+   */
+  valign?: "top" | "center" | "bottom";
+  /**
+   * Alignment of children along the horizontal axis
+   */
+  halign?: "left" | "center" | "right";
+  /**
+   * Allow for customer props due to ability to add
+   * custom breakpoints
+   */
+  [x: string]: any;
 }
 
-export interface HiddenProps {
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid. This enables debug mode for all the
-     * children of this component
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * Width during xs breakpoint.
-     */
-    xs?: boolean;
-    /**
-     * Width during sm breakpoint.
-     */
-    sm?: boolean;
-    /**
-     * Width during md breakpoint.
-     */
-    md?: boolean;
-    /**
-     * Width during lg breakpoint.
-     */
-    lg?: boolean;
+export interface BoxProps extends Styleable {
+  /**
+   * Draws all child columns with "bounding boxes" for easy
+   * visualization of the grid. This enables debug mode for all the
+   * children of this component
+   * @default false
+   */
+  debug?: boolean;
+  /**
+   * Controls the CSS flex property
+   */
+  flex?: string;
+  /**
+   * Sets whether the Box should fill up all available space
+   */
+  fill?: boolean;
+  /**
+   * Convenience property for disabling padding
+   */
+  fluid?: boolean;
+  /**
+   * Shifts the box to the right of the parent Bounds
+   */
+  shiftRight?: boolean;
+  /**
+   * Shifts the box to the ;eft of the parent Bounds
+   */
+  shiftLeft?: boolean;
+  /**
+   * Shifts the box to the top of the parent Bounds
+   */
+  shiftUp?: boolean;
+  /**
+   * Shifts the box to the bottom of the parent Bounds
+   */
+  shiftDown?: boolean;
+  /**
+   *  Allow for customer props due to ability to add
+   * custom breakpoints
+   */
+  [x: string]: any;
```

Comment:
Same question: why can this component take custom breakpoints? Didn't see it in the docs

Reply:
The breakpoints are passed down from the provider component. This is just allowing us to cover all custom breakpoints that are set.

[EG](https://github.com/garetmckinley/hedron/blob/master/stories/index.stories.js#L70)

Reply:
That's sad. It ruins the type safety of these components:(

Reply:
Not sure how to avoid it.

Reply:
Probably you can't. Nevermind, I'm just complaining out loud

Reply:
That's allowed 😄

Reply:
In any case. Maybe `any` could be changed to something like

```jsx
string | Styleable
```

?